wp-content/themes/pawar2018/js/scripts/app.js

```diff
@@ -17,6 +17,13 @@
   for (i = 0; i < accords.length; ++i) {
     let accordContent = accords[i].querySelector(".accordion-content");
     let accordTitle = accords[i].querySelector(".accordion-title");
+    if (window.location.hash) {
+      var hash = window.location.hash.substring(1);
```

Comment:
Could tighten this up a little by not re-declaring/assigning the `hash` variable for each pass through the loop. Could declare/assign prior, then just check the value in the loop.